tracks/rust/exercises/reverse-string/mentoring.md

```diff
@@ -0,0 +1,111 @@
+### Concepts
+
+- &str
+- String
+- Iterators
+- Iterator Methods
+
+### Reasonable solutions
+
+A reasonable solution should:
+
+- Use `.chars()`, `.rev()`, and `.collect()`
+- Be explicit with `.collect()` by using the turbofish operator `::<>`
+- Not use other iterators or iterator methods like `.into_iter()` or
+`.flat_map()`
+- Not use variables or loops
+- Not use `return`
+
+A bonus solution should:
+
+- Use the `unicode_segmentation` crate
+- Use `input.graphemes(true)` instead of
+`UnicodeSegmentation::graphemes(input, true)`
+
+### Examples
```

Comment:
Turn into level 4 heading (because these are examples of reasonable solutions)

Reply:
Done :)